docs/platform/howto/byoc/create-custom-cloud.md

```diff
@@ -769,6 +790,16 @@ Your new custom cloud is ready to use only after its status changes to
            --byoc-id "CUSTOM_CLOUD_IDENTIFIER" >| "tf_dir/tf_file.tf"
          ```
 
+         Replace the following:
+
+         - `ORGANIZATION_IDENTIFIER` with the identifier of your Aiven organization that
+           is to be connected with your own cloud account to create the custom cloud,
+           for example `org484338bebd2`. You can extract `ORGANIZATION_IDENTIFIER` from the
+           output of the `avn organization list` command.
+         - `CUSTOM_CLOUD_IDENTIFIER` with the identifier of your custom cloud, which you can
+           extract from the output of the `avn byoc create` command, for example
```

Comment:
I think `avn byoc list --organization-id ORGANIZATION_IDENTIFIER` could be recommended here, as it's possible to run that after the BYOC cloud has already been created. `byoc create` you only run once for a cloud, but may need the identifier at a later time too.

This identifier doesn't seem to be visible in console; I think we could consider exposing it in the console in BYOC cloud details for convenience.